front/lib/core_api.ts

```diff
@@ -134,10 +134,8 @@ export type CoreAPIDatabaseTable = {
 };
 
 export type CoreAPIDatabaseRow = {
-  created: number;
-  table_id: string;
```

Comment:
why were those removed?

Reply:
They were removed from the type in `core` because we want to reduce the # of types we have (want to use the same type for input and output when possible).
`created` wasn't really useful so it was easier to remove it both from in & out.
`table_id` as well